dask/base.py

```diff
@@ -648,14 +654,33 @@ def tokenize(*args, **kwargs):
     """
     if kwargs:
         args = args + (kwargs,)
-    return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
+    start_time = default_timer()
+    result = md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
+    time_taken = default_timer() - start_time
+    warn_config_key = "tokenize.warn_time_secs"
+    warn_time = config.get(warn_config_key, default=2)
```

Comment:
Please inline the literal above.  Also, see `dask.utils.parse_timedelta`.

Reply:
So, inline `warn_config_key` in  the warning message and in the call to `config.get`? I searched `parse_timedelta` and it doesn't appear to be used anywhere other than in its own tests. Would you prefer the config key to be `tokenize.warn_time` and then allow the user to specify units in the string value?

Reply:
parse_timedelta is used in distributed.

There we typically don't put the unit in the key, so just `tokenize.warn_duration` ir probably best.